api/types/types.go

```diff
@@ -263,6 +263,8 @@ type ContainerJSONBase struct {
 	ExecIDs         []string
 	HostConfig      *runconfig.HostConfig
 	GraphDriver     GraphDriverData
+	SizeRw          string
```

Comment:
these two fields should have the tag `json:",omitempty"` and the values should be only set when the flag is passed. Otherwise, the information is misleading. There could be containers with size 0(totally empty) but 0 wouldn't always mean that the container is empty, it could mean that the user didn't ask for the information. To avoid that confusion I'd rather have this logic:

1. Flag set: size values in the struct set, and the values are the real values.
2. Flag not set: not values in the struct.